src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -118,6 +118,14 @@
         <parameter key="sylius.based_on_id_invoice_number_generator.class" >Sylius\Component\Core\Payment\IdBasedInvoiceNumberGenerator</parameter>
 
         <parameter key="sylius.form.type.product_review.class">Sylius\Bundle\CoreBundle\Form\Type\ProductReviewType</parameter>
+
+        <parameter key="sylius.controller.sitemap.class">Sylius\Bundle\CoreBundle\Controller\SitemapController</parameter>
```

Comment:
Can we extract these services definitions to `sitemap.xml` or something like that?